tests/python/pants_test/backend/jvm/tasks/BUILD

```diff
@@ -466,7 +466,7 @@ python_tests(
     'testprojects/tests/java/org/pantsbuild/testproject:workdirs_directory',
   ],
   tags = {'integration'},
-  timeout = 240,
+  timeout = 1240,
```

Comment:
How'd you get to this number? Note that V2 does not yet respect timeouts, but will soon via https://github.com/pantsbuild/pants/pull/8278.

If it's timing out frequently, we could move it back to the V1 chroot runner, instead of V2 remote. To do this, add the target name to https://github.com/pantsbuild/pants/blob/master/build-support/ci_lists/integration_v2_blacklist.txt.

Reply:
I added a '1' to the digits that were there in an effort to get CI to pass last week. Not sure what the ideal timeout is.

Reply:
That bumps this test by a whole 16.6 minutes. For now, I'd keep the same as before since V2 ignores timeouts. Please do feel free to move it to the V2 blacklist I linked to above tho - I suspect this test performs much faster with V1 than V2 (this happens for several tests).